pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1621,6 +1625,18 @@ type ObjectEndpoints struct {
 	Secure []string `json:"secure"`
 }
 
+// ObjectStoreHostingSpec represents the hosting settings for the object store
+type ObjectStoreHostingSpec struct {
+	// A list of DNS names in which bucket can be accessed via virtual host path. These names need to valid according RFC-1123.
+	// Each domain requires wildcard support like ingress loadbalancer.
+	// Do not include the wildcard itself in the list of hostnames (e.g. use "mystore.example.com" instead of "*.mystore.example.com").
+	// Add all the hostnames including openshift routes to the list.
```

Comment:
Upstream users won't be using openshift routes.

```suggestion
	// Add all hostnames including user-created Kubernetes Service endpoints to the list.
```